js/helpers/Mediator.js

```diff
@@ -0,0 +1,25 @@
+export class Mediator {
+  subscribers = {};
+
+  subscribe = (event, callback) => {
+    this.subscribers[event] = this.subscribers[event] || [];
+    this.subscribers[event].push(callback);
+  };
```

Comment:
- Add empty line between statements, please.
- Do you know about `??` operator. You can use it here instead of `??` that is more suitable for this case. You can find details [here](https://javascript.info/nullish-coalescing-operator).